packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_HarmonicCoarseOperator_def.hpp

```diff
@@ -55,29 +55,29 @@ namespace FROSch {
     HarmonicCoarseOperator<SC,LO,GO,NO>::HarmonicCoarseOperator(ConstXMatrixPtr k,
                                                                 ParameterListPtr parameterList) :
     CoarseOperator<SC,LO,GO,NO> (k,parameterList),
-    ExtensionSolver_ (),
-    InterfaceCoarseSpaces_ (0),
-    AssembledInterfaceCoarseSpace_ (new CoarseSpace<SC,LO,GO,NO>(this->MpiComm_,this->SerialComm_)),
-    Dimensions_ (0),
```

Comment:
where is the information from 61 to 67 now?

Reply:
As in the other classes, I have moved this to the `*decl.hpp` file; see `packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_HarmonicCoarseOperator_decl.hpp `.